requirements.txt

```diff
@@ -1,2 +1,3 @@
 toml
 semver~=2.13
+six
```

Comment:
I'm slightly confused with this one: you clearly use `six` in the sources; still, it's not in the `requirements.txt`. Maybe I'm just missing some pythonesque feature, please forgive me then, I'm happy to revert.